### PR TITLE
Hide IR guidance on hub when survey is complete

### PR DESCRIPTION
**Pull request by berroar**
### What is the context of this PR?

Individual Response guidance is now hidden on the hub when the survey is complete.

### How to review 

Use the `test_individual_response` schema to check that the IR guidance is not displayed on the hub when the survey is complete

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by berroar** [comment]
[Changes to hub at submission point (s)](https://trello.com/c/nWy4e24I/4277-changes-to-hub-at-submission-point-s)

File: templates/hub.html [hunk 1]
```diff
@@ -35,7 +35,7 @@ <h1 class="u-mt-m">{{ content.title }}</h1>
         <p class="u-mb-no" data-qa="guidance">{{ content.guidance }}</p>
       </div>
     {% endif %}
-    {% if content.individual_response_url %}
+    {% if content.individual_response_url and not content.survey_complete %}
```
Comment (MebinAbraham) [inline templates/hub.html:38]: What if `individual_response_url` wasn't set when survey was complete?

Comment (LJBabbage) [reply templates/hub.html:38]: As survey complete is only used for IR at the moment, and it adds logic in the template I think not setting  individual_response_url makes more sense. However not something I will block on

**Review by LJBabbage — APPROVED** [review]

File: templates/hub.html (hunk 1 above)
Comment (berroar) [reply templates/hub.html]: I've updated it now as probably better to use less logic

**Review by MebinAbraham — APPROVED** [review]

**Pull request merged**